download-libs.sh

```diff
@@ -25,12 +25,12 @@ cd lib
 
 # checkout closure library
 if [ ! -d closure-library/.git ]; then
-  git clone --depth 1 https://github.com/google/closure-library/ closure-library
+  git clone https://github.com/google/closure-library
```

Comment:
We used `--depth 1` to speedup the `install_deps` step. Any reason to download the whole repo history of dependencies? If someone's hacking around with different versions of the deps, he may download the history manually.